integrations/clickup/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,67 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: team
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupTeam"'
+          identifier: .id
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .id + \"/home\""
+            members: '[.members[].user.email]'
+  - kind: space
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupSpace"'
+          identifier: .id
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .__team.id + \"/v/o/s/\" + .id"
+            statuses: .statuses
+            archived: .archived
+            private: .private
+          relations:
+            team: .__team.id
+  - kind: project
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupProject"'
+          identifier: .id
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .__team.id + \"/v/b/li/\" + .id"
+            startDate: if .start_date!=null then .start_date | tonumber / 1000 | todate else "" | todate end
+            endDate: if .due_date!=null then .due_date | tonumber / 1000 | todate else "" | todate end
+            totalIssues: .task_count
+          relations:
+            space: .space.id
+  - kind: issue
+    selector:
+      query: 'true'
+    port:
```

Comment:
why we call it `issue` if all the clickup reference calls it `task`?

Reply:
Done